packages/web-api-scan-job-manager/src/setup-web-api-scan-job-manager-container.ts

```diff
@@ -19,11 +19,9 @@ export function setupWebApiScanJobManagerContainer(): Container {
         .inSingletonScope();
 
     container
-        .bind(ScanTaskParameterProvider)
-        .toSelf()
+        .bind<BatchTaskParameterProvider>(AzureServicesIocTypes.BatchTaskParameterProvider)
+        .to(ScanTaskParameterProvider)
```

Comment:
can u update the ut in setup-web-api-scan-job-manager-container.spec.ts to also validate singleton resolution for this type under "resolves service config to singleton value" test?